dbms/src/AggregateFunctions/AggregateFunctionTopK.h

```diff
@@ -132,8 +132,8 @@ struct AggregateFunctionTopKGenericData
         std::string,
         StringRef,
         StringRefHash,
-        HashTableGrower<TOP_K_DEFAULT_DEGREE>,
-        HashTableAllocatorWithStackMemory<sizeof(StringRef) * (1 << (TOP_K_DEFAULT_DEGREE/2))>
+        HashTableGrower<4>,
+        HashTableAllocatorWithStackMemory<sizeof(StringRef) * (1 << 4)>
```

Comment:
But `sizeof(Cell)` is larger than just key.
PS. You may expose cell type from HashTable, if you wish.